types/twine-sugarcube/test/extensions.ts

```diff
@@ -7,6 +7,18 @@ n = ar1.count(1, 1);
 // @ts-expect-error
 n = ar1.count('a', 1);
 
+declare function countPredicate(fruit: string, index: number, array: string[]): boolean;
+const fruits = ["Apples", "Oranges", "Plums", "Oranges"];
+fruits.countWith(countPredicate);
+const fakeThis = {
+	propA: "string"
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/twine-sugarcube/test/extensions.ts;
  Line: 14;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.